content/blogposts/2022/cloud-managed-instances-for-enterprise.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Secure and scalable Sourcegraph Cloud managed instances for the enterprise"
+publishDate: 2022-09-27T10:00-07:00
+description: Sourcegraph Cloud, the single-tenant, dedicated Cloud version of Sourcegraph, is now generally available.
+tags: [blog]
+slug: "enterprise-cloud"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+---
+
+Soucegraph has long existed as self-hosted software, letting our customers confidently and securely run Sourcegraph in their own environments. After almost a decade of developing self-hosted Sourcegraph, we’ve gained the trust of some of the most sophisticated companies and development teams in the world, allowing us to work with their most sensitive IP: their code.
```

Comment:
"we’ve gained the trust of some of the most sophisticated companies and development teams in the world"

Can we add some company names here so it carries more credibility?